patches/minecraft/net/minecraft/network/play/server/SUpdateRecipesPacket.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/network/play/server/SUpdateRecipesPacket.java
++++ b/net/minecraft/network/play/server/SUpdateRecipesPacket.java
+@@ -46,7 +_,7 @@
+ 
+    }
+ 
+-   @OnlyIn(Dist.CLIENT)
++   //@OnlyIn(Dist.CLIENT)
```

Comment:
I'm pretty sure you can't do this.

Reply:
Forge has a side only annotation stripper if I remember, I don't think commenting out only in is the way they do it

Reply:
Oh, didn't know it existed. I will change it.